sdk/storage/azure-storage-common/inc/azure/storage/common/internal/xml_wrapper.hpp

```diff
@@ -19,62 +20,54 @@ namespace Azure { namespace Storage { namespace _internal {
 
   struct XmlNode final
   {
-    explicit XmlNode(XmlNodeType type, std::string name = std::string())
-        : Type(type), Name(std::move(name))
-    {
-    }
-
-    explicit XmlNode(XmlNodeType type, std::string name, std::string value)
-        : Type(type), Name(std::move(name)), Value(std::move(value)), HasValue(true)
+    explicit XmlNode(
+        XmlNodeType type,
+        std::string name = std::string(),
```

Comment:
```suggestion
        std::string const& name = {},
```
Use initializer syntax instead of explicitly setting the name. Also always pass structures by `const&` unless absolutely required.